tests/cpp/test_matmul.cpp

```diff
@@ -3154,6 +3154,8 @@ TEST_P(MatmulTestWithLayout, AmpereMatmulBatchSplitKBias) {
 
 // Avoid lowering error https://github.com/NVIDIA/Fuser/issues/1808
 TEST_F(MatmulTest, ReproIssue1808) {
+  NVFUSER_TEST_CUDA_ARCH_RANGE_GUARD(8, 0, 9, 0);
```

Comment:
```suggestion
  // TODO: Make these tests work with Hopper as well as Ampere
  NVFUSER_TEST_CUDA_ARCH_RANGE_GUARD(8, 0, 9, 0);
```

Reply:
I added this comment to the guards in `test_matmul_scheduler`. That test used specific ampere configuration.